command_abstraction.py

```
@@ -102,58 +102,59 @@ async def create_thread(self, name: str) -> Optional[discord.Thread]:
-                    fetched_message = await self.channel.fetch_message(message.id)
-                    return await fetched_message.create_thread(name=name)
-                except (discord.HTTPException, discord.NotFound) as fetch_error:
-                    logger.warning(f"Failed to fetch message {message.id} for thread creation: {fetch_error}")
-                    # Fallback to channel thread creation
-                    return await self.create_thread(name)
-            else:
-                logger.error(f"ValueError creating thread from message '{name}': {e}")
-                return None
+                logger.info(f"Message lacks guild info, falling back to channel thread: {e}")
+                return await self.create_thread(name)
+            logger.error(f"ValueError creating thread from message '{name}': {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)